services/director-v2/tests/unit/with_dbs/conftest.py

```diff
@@ -265,27 +273,31 @@ async def running_project(
     user = create_registered_user()
     created_project = await project(user, workbench=fake_workbench_without_outputs)
     now_time = arrow.utcnow().datetime
+    created_tasks = await create_tasks(
+        user=user,
+        project=created_project,
+        state=StateType.RUNNING,
+        progress=0.0,
+        start=now_time,
+    )
+    tasks_to_run = [t for t in created_tasks if t.node_class == NodeClass.COMPUTATIONAL]
+
     return RunningProject(
         user=user,
         project=created_project,
         pipeline=await create_pipeline(
             project_id=f"{created_project.uuid}",
             dag_adjacency_list=fake_workbench_adjacency,
         ),
-        tasks=await create_tasks(
-            user=user,
-            project=created_project,
-            state=StateType.RUNNING,
-            progress=0.0,
-            start=now_time,
-        ),
+        tasks=created_tasks,
         runs=await create_comp_run(
             user=user,
             project=created_project,
             started=now_time,
             result=StateType.RUNNING,
         ),
```

Comment:
I think here there is something wrong. create_comp_run should get the same dag that you pass `create_pipeline`